geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -17,47 +17,105 @@ instructions:
       To use, type `geoips config install <test_dataset_name>`.
     output_info:
       - Not Applicable
-  geoips_get:
+  geoips_get: &geoips-get-interface
     help_str: |
-      Retrieve an appropriate GeoIPS artifact. Currently supported `get` calls are:
-      ["family", "interface", "package", "plugin"]
+      Retrieve an the selected GeoIPS artifact. For this command, an artifact is one of
+      ['interface', 'plugin', 'family']. To get more information on how to retrieve
+      these artifacts, see below.
```

Comment:
Grammar:
> Retrieve information about a GeoIPS artifact.

Reply:
I think we should also consider how this can be made more specific to each sub-command so that, for example, `geoips get algorithm -h` can show something specific to algorithms.

Reply:
Fixed!

Reply:
I'll make an issue for making the help information more specific about the command ran. I think that's going to require a decent amount of testing and refactoring and should likely be put in a different PR.

Reply:
See https://github.com/NRLMMD-GEOIPS/geoips/issues/645